### PR TITLE
OrderBook: update events

### PR DESCRIPTION
**Pull request by wer1st**

### CODE REVIEW

File: pallets/order-book/src/lib.rs [hunk 1]
```diff
@@ -346,14 +350,23 @@ pub mod pallet {
             order_id: T::OrderId,
             owner_id: T::AccountId,
             side: PriceVariant,
+            price: OrderPrice,
             amount: OrderAmount,
         },
 
+        /// All amount of the limit order is executed
+        LimitOrderFilled {
```
Comment (bragov4ik) [inline pallets/order-book/src/lib.rs:358]: `LimitOrderFulfilled` seems more accurate/readable, do you agree?

File: pallets/order-book/src/types.rs [hunk 2]
```diff
@@ -67,6 +67,17 @@ pub enum OrderBookStatus {
     Stop,
 }
 
+#[derive(
+    Encode, Decode, PartialEq, Eq, Copy, Clone, Debug, scale_info::TypeInfo, MaxEncodedLen,
+)]
+pub enum CancelReason {
+    /// User cancels the limit order by themselves
```
Comment (bragov4ik) [inline pallets/order-book/src/types.rs:74]:
```suggestion
    /// User cancels the limit order by themself
```
minor correction 🙃

**Review by bragov4ik — APPROVED** [review]
LGTM, but I want to clarify:
there's a point in the task that says
- `LimitOrderIsSplitIntoMarketOrderAndLimitOrder` should go together with `LimitOrderPlaced` 

Is it addressed here? I just didn't quite catch that

File: pallets/order-book/src/lib.rs (hunk 1 above)
Comment (wer1st) [reply pallets/order-book/src/lib.rs:358]: Not. `Filled` is a common status for different exchanges

File: pallets/order-book/src/types.rs (hunk 2 above)
Comment (wer1st) [reply pallets/order-book/src/types.rs:74]: Both variant are ok, because of moder notation. But ok

**Comment by wer1st** [comment]
> LGTM, but I want to clarify: there's a point in the task that says
> 
> * `LimitOrderIsSplitIntoMarketOrderAndLimitOrder` should go together with `LimitOrderPlaced`
> 
> Is it addressed here? I just didn't quite catch that

It already works

**Review by vovac12 — APPROVED** [review]

**Pull request merged**